vmsdk/python/tests/tdx/test_sdk.py

```diff
@@ -0,0 +1,115 @@
+        assert event_logs is not None
+        for event in event_logs:
+            if isinstance(event, TcgImrEvent):
+                sha384_algo = sha384()
+                sha384_algo.update(rtmrs[event.imr_index] + event.digests[0].hash)
+                rtmrs[event.imr_index] = sha384_algo.digest()
+        return rtmrs
+
+    def check_imr(self, imr_index: int, alg_id: int, rtmr: bytes):
+        """Check individual IMR.
+        Compare the 4 IMR hash with the hash derived by replay event log. They are
+        expected to be same.
+        Args:
+            imr_index: an integer specified the IMR index.
+            alg_id: an integer specified the hash algorithm.
+            rtmr: bytes of RTMR data for comparison.
+        """
+        assert 0 <= imr_index < TestCCTrustedVmSdkTdx.MEASUREMENT_COUNT
+        assert rtmr is not None
+        assert alg_id == TcgAlgorithmRegistry.TPM_ALG_SHA384
+        imr = CCTrustedVmSdk.inst().get_measurement([imr_index, alg_id])
+        assert imr is not None
+        digest_obj = imr.digest(alg_id)
+        assert digest_obj is not None
+        digest_alg_id = digest_obj.alg.alg_id
+        assert digest_alg_id == TcgAlgorithmRegistry.TPM_ALG_SHA384
+        digest_hash = digest_obj.hash
+        assert digest_hash is not None
+        assert digest_hash == rtmr, \
+            f"rtmr {rtmr.hex()} doesn't equal digest {digest_hash.hex()}"
+
+    def test_get_measurement(self):
+        """Test measurement result.
+        The test is done by compare the measurement register against the value
+        derived by replay eventlog.
+        """
+        cmdline = None
+        try:
+            with open(TestCCTrustedVmSdkTdx.PROC_CMDLINE, "r", encoding="utf-8") as proc_cmdline:
+                cmdline = proc_cmdline.readline()
+        except (PermissionError, OSError):
+            LOG.error("Need root permission to open file %s", TestCCTrustedVmSdkTdx.PROC_CMDLINE)
+            assert False
+        assert cmdline is not None
+
+        ima_policy = None
+        if "ima_hash=sha384" in cmdline:
+            m = re.search(r".*ima_policy=(\w+).*", cmdline)
+            if m is not None:
+                ima_policy = m.group(1)
+            LOG.info("ima_policy is %s", ima_policy)
+            if (ima_policy is None or
+                ima_policy in TestCCTrustedVmSdkTdx.IMA_SUPPORTED_POLICIES):
+                alg = CCTrustedVmSdk.inst().get_default_algorithms()
+                rtmrs = self.replay_eventlog()
+                self.check_imr(0, alg.alg_id, rtmrs[0])
+                self.check_imr(1, alg.alg_id, rtmrs[1])
+                self.check_imr(2, alg.alg_id, rtmrs[2])
+                self.check_imr(3, alg.alg_id, rtmrs[3])
+        else:
+            LOG.info("IMA is not enabled! Unable to compare with event log!")
+
+    def test_get_eventlog(self):
+        """Test get_eventlog result."""
+        #TODO: verify the eventlog value.
```

Comment:
Please remove the test if not implemented yet, otherwise the blank test will fail to run, correct?

Reply:
Please let me remove it to avoid confusion (although it doesn't fail the test). I'll add them back when the TODO is done. Thanks!